src/main/java/org/example/datastructures/binarysearchtree/n-ary Pre  order  Traversal.java

```diff
@@ -0,0 +1,25 @@
+class Solution {
```

Comment:
File name should not contain spaces or special character. It should in CamelCase.